features/topic_id_content.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Behaviour of application when doing the GET /topics/{id}/content endpoint, using a stripped down version of the database
```

Comment:
the db setup for these tests are identical, so we can move the **Given I have these topics:** and **And I have these contents:** into a background block at the top of the file, only needing to declare them once instead of twice

Reply:
Done